docs/source/starters/new_project_tools.md

```diff
@@ -1,4 +1,199 @@
-# Configure a new project
+# Using tools to configure your Kedro project
 
-<!--TO DO-->
-<!--Detailed usage of add-ons goes here-->
+## Configuring your tool selection
+
+There are several ways to configure your project tool when creating a new Kedro project.
```

Comment:
It sounds a bit odd that you can "configure your project tool" I'd phrase it as "configure your project setup" or "configure your project template"

Reply:
Changed to "Configure your project setup" in [0f1cbf4](https://github.com/kedro-org/kedro/pull/3331/commits/0f1cbf4dae8ac118f4a408f592aa2b9df18a838f)